src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicListener.cs

```diff
@@ -186,7 +193,7 @@ internal unsafe uint ListenerCallbackHandler(ref ListenerEvent evt)
 
         private void StopAcceptingConnections()
         {
-            _acceptConnectionQueue.Writer.TryComplete();
+            _acceptConnectionQueue?.Writer.TryComplete();
```

Comment:
It looks like _acceptConnectionQueue is always assigned in the only constructor.

```suggestion
            _acceptConnectionQueue.Writer.TryComplete();
```